libertai_client/commands/agent.py

```diff
@@ -97,6 +97,14 @@ async def deploy(
             prompt=False,
         ),
     ] = None,
+    format: Annotated[
+        bool | None,
+        typer.Option(
+            help="Output format for the deployment information",
+            case_sensitive=False,
+            prompt=False,
+        ),
+    ] = None,
```

Comment:
it's still None potentially, use the link I said to make it a bool without the counter option `--no-format`